.travis.yml

```diff
@@ -0,0 +1,47 @@
+language: java
+os: linux
+
+dist: xenial
+jdk: openjdk8
+
+sudo: required
+services:
+  - docker
+
+before_install:
+- cd ${TRAVIS_BUILD_DIR}/..
+- git clone --depth=50 --branch=master https://github.com/vlingo/vlingo-common.git
+- cd vlingo-common && mvn install -DskipTests=true
+- cd ${TRAVIS_BUILD_DIR}/..
+- git clone --depth=50 --branch=master https://github.com/vlingo/vlingo-actors.git
+- cd vlingo-actors && mvn install -DskipTests=true
+- cd ${TRAVIS_BUILD_DIR}/..
+- git clone --depth=50 --branch=master https://github.com/vlingo/vlingo-wire.git
+- cd vlingo-wire && mvn install -DskipTests=true
+- cd ${TRAVIS_BUILD_DIR}/..
+- git clone --depth=50 --branch=master https://github.com/vlingo/vlingo-cluster.git
+- cd vlingo-cluster && mvn install -DskipTests=true
+- cd ${TRAVIS_BUILD_DIR}/..
+- git clone --depth=50 --branch=master https://github.com/vlingo/vlingo-symbio.git
+- cd vlingo-symbio && mvn install -DskipTests=true
+- cd ${TRAVIS_BUILD_DIR}/..
+- git clone --depth=50 --branch=master https://github.com/vlingo/vlingo-lattice.git
+- cd vlingo-lattice && mvn install -DskipTests=true
+- cd ${TRAVIS_BUILD_DIR}
+script:
+- mvn install
+
+deploy:
+  provider: bintray
+  file: bintray.json
+  skip_cleanup: true
+  user: vaughnvernon
+  on:
+    branch: master
+  key:
+    secure: iyHRwuxdPG7asRlnkEnrGxb8+ZOYZV9Vz2QCqKOlHk4QR48nDvoD3msX0AGXYOAbXCuSdweADP7NIe0LhqZ9D6JLvmdnJWXRjAvfook1Zq4J/7j4hkthL63g2qZ8VRIEi6A8XJRHC1SM+OqFCrsPJJjr23+OB5YTqxdSkHcbHUupPwMxI1U6ScL+3KORi1Qk0bmTllC9VL5hL8U3DyujHyyLFow9zdS29mO9S74CC59OQidTRPDBVWkZqOKpgfQcK4AHAnFffxZFJLxWP+YKst1u66DNfUUxx26Qa0XfmHwo16bbZqGMLmd9WD245HDZ8vTKwm14HVf9ZfIVxPYbZTNW/egxYOCjJW1U4ofOS5/TpAoHpzko+Bb4rvIEzSeyooVovv54pa4/9Kh7vuAsk3tgu6TMJhTwj5X0bAyKyhgDsZJNGRoSKj64bqSPTvsZTFIZ2Vg+OXKGmDLh7e9UnElAq/vnESs2c1suTosVB1QHCGwiKpMZIh9ej+4HTc9UTga6jvJxGd6lLTa9Te0c7+wZEJg+SV1wnYfXuMRG6Lxaf7IhzhFUIp4n3IMVakvHlPCoVzIq/CHknqAr0310gsYTGAVteCLzWokvx9PnCBEBkmDhVPIvecpKuY2kLkX9eK0L8OL9529GNIvt/jvVgDtjpLF/tS+fdxKSyUN2o3o=
```

Comment:
@alexguzun Does this key work or do I need to generate one? I will merge and check the outcome and generate a new key if needed.

Reply:
Not sure. I have copied it from other project.